src/main/java/cholog/supp/common/config/JpaAuditingConfig.java

```diff
@@ -0,0 +1,10 @@
+package cholog.supp.common.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
+
+@Configuration
+@EnableJpaAuditing
+public class JpaAuditingConfig {
+
+}
```

Comment:
```suggestion
}

```
여기도 EOL(End Of Line) 놓쳤다!!